package/rpm/install_section

```diff
@@ -0,0 +1,5 @@
+python3 setup.py install --single-version-externally-managed -O1 --root=%{buildroot} --record=INSTALLED_FILES
+
+# make sure we also own the top level and egg-info raw dir to make things easy on uninstall
+echo "%{python3_sitelib}/%{name}-*egg-info" >> INSTALLED_FILES
+echo "%{python3_sitelib}/%{name}" >> INSTALLED_FILES
```

Comment:
Can you add the missing end line marker?

Reply:
I added the missing end line.